TODO.txt

```diff
@@ -0,0 +1,42 @@
+A lot of needed fixes are similar one to another. Would be great to have generic code that could be reused for similar fixres.
```

Comment:
Should probably be `fixers` instead of `fixres`.

Reply:
probably ;) let us not focus on typos in internal todo list that will be drop when finishing the PR ;)